adds example using digitalocean spaces tutorial (#1872)
iterative/dvc.org: content/docs/command-reference/remote/modify.md

  > - [ACL Overview - Permissions](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#permissions)
  > - [Put Object ACL](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html)

S3 remotes can also be configured entirely via environment variables:

```dvc

> $ export AWS_ACCESS_KEY_ID='<my-access-key>'

Reviewer: I agree about moving this to the main S3 section.

  > - [ACL Overview - Permissions](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#permissions)
  > - [Put Object ACL](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html)

> 

Reviewer: ```suggestion
```

                    https://object-storage.example.com
```

Besides that, any settings that are available for Amazon S3 (see previous
section) may be available for S3 compatible storage. For example, let's setup a
DVC remote using the `example-name`
[DigitalOcean space](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key)
(equivalent to a bucket in AWS) in the `nyc3` region:

```dvc
$ dvc remote add -d myremote s3://example-name/path/to/use

> $ dvc remote modify myremote https://nyc3.digitaloceanspaces.com

Reviewer: ```suggestion
$ dvc remote modify myremote endpointurl \
                             https://nyc3.digitaloceanspaces.com
```

                    https://object-storage.example.com
```

> Besides that, any settings that are available for AWS S3 and described above, are
> available for an S3 compatible storage if it supports them.
> 
> For example, the
> [tutorial for DigitalOcean spaces](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key)
> creates a space (equivalent to an AWS bucket) called `example-name` in the `nyc3`
> region. Your remote settings would be set like this: 

Reviewer: ```suggestion
Besides that, any settings that are available for Amazon S3 (see previous
section) may be available for S3 compatible storage. For example, let's setup a
DVC remote using the `example-name`
[DigitalOcean space](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key)
(equivalent to a bucket in AWS) in the `nyc3` region:
```
